ClosedXML_Tests/TestHelper.cs

```diff
@@ -101,7 +101,7 @@ public static void RunTestExample<T>(string filePartName, bool evaluateFormulae
             }
         }
 
-        public static void CreateAndCompare(Func<IXLWorkbook> workbookGenerator, string referenceResource, bool evaluateFormulae = false)
+        public static void CreateAndCompare(Func<IXLWorkbook> workbookGenerator, string referenceResource, bool evaluateFormulae = false, SaveOptions options = null)
```

Comment:
Once `options` has been added there is no sense in `evaluateFormulae` argument. Remove it.

Reply:
On the other hand, if you get rid of CreateAndCompare tests you added, this method may be reverted.

Reply:
test implementation adjusted - no createAndCompare used anymore